velox/exec/tests/ContainerRowSerdeTest.cpp

```diff
@@ -35,6 +35,26 @@ class ContainerRowSerdeTest : public testing::Test,
     return position;
   }
 
+  std::vector<HashStringAllocator::Position> serializeWithPositions(
+      const VectorPtr& data) {
+    std::vector<HashStringAllocator::Position> positions;
+    auto size = data->size();
+    positions.reserve(size);
+
+    for (auto i = 0; i < size; ++i) {
+      ByteStream out(&allocator_);
+      auto position = allocator_.newWrite(out);
+      ContainerRowSerde::serialize(*data, i, out);
+      allocator_.finishWrite(out, 0);
+
+      auto cur = std::make_shared<ByteStream>();
```

Comment:
The logic using 'cur' is not needed. Let's remove.

Reply:
Fixed